build-tools/helm/spark-kubernetes-operator/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
```

Comment:
Do we need `.svn` and `.hg`?

Reply:
We do not - it's a blanket ignore. Lemme remove this unused. Thanks for the catch